ec2-find

```diff
@@ -207,6 +213,10 @@ while [[ $# -gt 0 ]]; do
       ip_only="yes"
       use_headers="no"
     ;;
+    -i|--ids-only)
```

Comment:
I think the option long-form should be "--id-only" so it's consistent with "--ip-only".

Reply:
You also need to update the help message.